test/functional/shared/resources/testdata/corerp-resources-terraform-redis.bicep

```diff
@@ -21,7 +21,7 @@ resource env 'Applications.Core/environments@2023-10-01-preview' = {
       'Applications.Core/extenders': {
         default: {
           templateKind: 'terraform'
-          templatePath: '${moduleServer}/kubernetes-redis.zip'
+          templatePath: '${moduleServer}/kubernetes-redis.zip//kubernetes-redis/modules'
```

Comment:
Looking at the docs https://docs.python.org/3/library/shutil.html#shutil-archiving-example-with-basedir, I think unpack returns directories inside root directory. So we either change the path here to `//modules` or specify base directory while creating the zip here: https://github.com/radius-project/radius/blob/main/.github/scripts/publish-test-terraform-recipes.py#L63 - root dir would change to tmp_dir and base directory would be recipe_dir

Reply:
Updated to `//modules` and that seems to have fixed the path issue. Thanks for finding this!